### PR TITLE
[tests-only]Add tests for opening edit panel via quick action and context menu

### PR DESCRIPTION
**Pull request by SwikritiT**
## Description
This Pr adds tests for opening the edit panel via quick action and context menu

## Related Issue

- part of: https://github.com/owncloud/web/issues/8601

## How Has This Been Tested?

- locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/e2e/cucumber/steps/ui/adminSettings.ts [hunk 1]
```diff
@@ -432,3 +445,27 @@ Then(
...
+    switch (action) {
+      case 'quick action':
+        await usersObject.openEditPanel({ key: actionUser, action: 'quick-action' })
+        break
+      case 'context menu':
+        await usersObject.openEditPanel({ key: actionUser, action: 'context-menu' })
+        break
+      default:
+        throw new Error(`${action} not implemented`)
+    }
```
Comment (saw-jan) [inline tests/e2e/cucumber/steps/ui/adminSettings.ts:463]:
```suggestion
    await usersObject.openEditPanel({ key: actionUser, action: action.replace(' ', '-') })
```

File: tests/e2e/support/objects/app-admin-settings/users/index.ts [hunk 2]
```diff
@@ -96,4 +123,13 @@ export class Users {
   async deleteUserUsingBatchAction(): Promise<void> {
     await deleteUserUsingBatchAction({ page: this.#page })
   }
+
+  async openEditPanel({ key, action }: { key: string; action: string }): Promise<void> {
+    const { uuid } = this.#usersEnvironment.getUser({ key })
+    await openEditPanel({ page: this.#page, uuid, action })
+  }
+
+  async editPanelVisible(): Promise<void> {
```
Comment (saw-jan) [inline tests/e2e/support/objects/app-admin-settings/users/index.ts:132]:
```suggestion
  async waitForEditPanelToBeVisible(): Promise<void> {
```

File: tests/e2e/cucumber/features/smoke/admin-settings/users.ocis.feature [hunk 3]
```diff
@@ -152,3 +152,18 @@ Feature: users management
...
+      | Alice |
+      | Brian |
+      | Carol |
+    And "Admin" logs in
+    And "Admin" opens the "admin-settings" app
+    And "Admin" navigates to the users management page
+    When "Admin" opens the edit panel of user "Brian" using the quick action
+    Then "Admin" should see the edit panel
+    When "Admin" opens the edit panel of user "Brian" using the context menu
+    Then "Admin" should see the edit panel
```
Comment (ScharfViktor) [inline tests/e2e/cucumber/features/smoke/admin-settings/users.ocis.feature:169]: duplicate actions

Comment (SwikritiT) [reply tests/e2e/cucumber/features/smoke/admin-settings/users.ocis.feature:169]: no actually the tests is to check if the edit panel can be opened using the `quick action` and `context menu`

Comment (ScharfViktor) [reply tests/e2e/cucumber/features/smoke/admin-settings/users.ocis.feature:169]: yes, sorry

**Review by ScharfViktor — APPROVED** [review]

**Review by saw-jan — COMMENTED** [review]
LGTM

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/34044/12/1
:boom: The e2e-ocis tests pipeline failed. The build has been cancelled.

**Pull request merged**